README.md

```diff
@@ -112,6 +112,27 @@ install
 
 Please refer to the CMake Manual for more details.
 
+**Windows**:
+Different configurations have different run-time requirements.  The `Debug`
+configuration uses the windows debug C runtime, whereas the `RelWithDebInfo`
+does not.  This may determine which version you can use with your executable.
+
+Some special notes regarding `cmake`:
+- cmake doesn't need the configuration int the `-B` step since it creates
+  all the `*.vcxproj` files at once.
+- for `cmake --build`, **do no** specify `--parallel`, but **do** specify your
+  `--config RelWithDebInfo` or whichever configuration is appropriate.
+
+In short, do something like
+```
+> cmake -B build
+> cmake --build build --config RelWithDebInfo
+> cmake --install build --prefix install --config RelWithDebInfo
+```
+
+In addition to the optional `winhttp.lib`, windows builds require linking
+with `version.lib` and `dbghelp.lib`
```

Comment:
Already handled in https://github.com/getsentry/sentry-native/blob/master/CMakeLists.txt#L343-L366

Reply:
Right, but it probably ought to be pointed out to users of the `sentry.lib` library, who otherwise need to hunt through their linker errors to figure out which windows libraries to include.  It is already mentioned that `winhttp.lib` is a dependency but the others aren't mentioned.